featuretools/entityset/entityset.py

```diff
@@ -1619,8 +1619,6 @@ def query_by_values(
         if not column_name:
             column_name = dataframe.ww.index
 
-        instance_vals = _vals_to_series(instance_vals, column_name)
```

Comment:
I suspect we need this function call still. I think within the call, we can remove the check for `pd.dataframe`.